### PR TITLE
[feature] Add liquidity module and cover tests. 

### LINKED ISSUE #6: [Feature Request] Missing functions
We have to implement the following funcs:

* Create Pool.
* Remove liquidity.
* Add liquidity.
* Quotes before add liquidity or burn liquidity.
* Check if pool ever exists.

### PR DESCRIPTION
**Pull request by lusd**
Fixes #6 

> It's a good idea to open an issue first for discussion.

- [ ] Tests pass
- [ ] Appropriate changes to README are included in PR

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -231,4 +231,11 @@ const decimalValue2 = convertValueToDecimal('0.005', 8); // 500000 (0.005 coin w
 })()
 ```
 
+### You want to calculate rate /add pool liquidity for a pair of tokens
+```typescript
+(async () => {
+
```
Comment (sirWill) [inline README.md]: Better to add a comment here about go looking on test file and "an example will be available soon", than leave an empty function call.

**Review by sirWill — APPROVED** [review]
